### ISSUE TITLE
Bootstrap.sh fails on xUbuntu 15.04

### ISSUE THREAD

**Issue by cyocum** [simoncozens/sile#168]
Directly after git clone run:

./bootstrap.sh

output is:

```
./bootstrap.sh: 8: ./bootstrap.sh: [[: not found
libtoolize: putting auxiliary files in `.'.
libtoolize: copying file `./ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
libtoolize: copying file `m4/ltsugar.m4'
libtoolize: copying file `m4/ltversion.m4'
libtoolize: copying file `m4/lt~obsolete.m4'
configure.ac:6: installing './compile'
configure.ac:6: installing './config.guess'
configure.ac:6: installing './config.sub'
configure.ac:3: installing './install-sh'
configure.ac:3: installing './missing'
src/Makefile.am: installing './depcomp'
autoreconf: 'configure.ac' or 'configure.in' is required
```

expected output is the submodules are cloned.

The fix here seems to be changing "#!/usr/bin/env sh" to "#!/bin/bash".  I'm not sure that this will work everywhere though.

**Comment by alerque** [comment]
Thanks for the report. The `[[` test syntax is indeed a bash thing that many sh emulators are not equipped to handle, so the hash bang line is wrong. That being said I think the correct fix is to make it `#!/usr/bin/env bash`. Can you confirm that this works on your system? Usually that syntax is considered the most portable way to handle the situation, but it does need to be bash not sh.

**Comment by cyocum** [comment]
Yep, that seems to work here is a patch:

```
diff --git a/bootstrap.sh b/bootstrap.sh
index 55a8ba3..c0d4e20 100755
--- a/bootstrap.sh
+++ b/bootstrap.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 # We neet a local copy of the libtexpdf library to compile. If this was
 # downloaded as a src distibution package this will exist already, but if not
```

**Comment by simoncozens** [comment]
This should now be fixed, thanks to #169.
